docs/index.md

```diff
@@ -1063,6 +1064,12 @@ tcp://192.168.99.109:2376
 #### Amazon Web Services
 Create machines on [Amazon Web Services](http://aws.amazon.com).  You will need an Access Key ID, Secret Access Key and a VPC ID.  To find the VPC ID, login to the AWS console and go to Services -> VPC -> Your VPCs.  Select the one where you would like to launch the instance.
 
+Use `amazonec2` for the `--driver` and include the 3 required parameters. Depending upon the configured VPC, you may have to specify `--amazonec2-zone` if a subnet does not exist in Availability Zone `a`.
```

Comment:
re VPC: The VPC id is in the ID column. for step 3, it should say: Use an ID specified in the ID column
re Availability Zone: `a` is used by default. To see if you have resources in availability zone `a`:
1.  Login to the AWS console
2.  Go to **Services -> VPC -> Subnets**
3.  Locate your selected VPC ID from the VPC column. Examine the `Availability Zone` column to see if `a` exists matching the VPC ID, such as `us-east1-a`. If `a` doesn't exist then you must specify `--amazonec2-zone` during creation such as `--amazonec2-zone c`. Or create a new subnet residing on Availability Zone `a`.